### ISSUE TITLE
Tasks created by importing local storage are not sorted -- when using IO for external storage

### ISSUE THREAD

**Issue by mikbuch** [heartexlabs/label-studio#2266]
**Describe the bug**
When files from the local storage (file system) are synchronized with label-studio, using IO external storage (i.e., added to the project and tasks), the tasks for these files are in random order. In other words, the IDs of the tasks have nothing to do with either filename of the original file, not the time when this file was created in the file system.

**To Reproduce**
1. Prepare files in the file system
2. Synchronize/add files to the label-studio using IO Storage API (local files)
3. Go to a project
4. See that the IDs for tasks are random

**Expected behavior**
The IDs of the tasks should be sorted by filenames of the files to be added to the label-studio.

**Screenshots**
n/a -- can provide if really needed

**Environment (please complete the following information):**
 - OS: Ubuntu and macOS
 - Label Studio Version Release 1.4.1

**Additional context**

In the `label_studio/io_storages/localfiles/models.py`

there is:

```
for file in path.rglob('*'):
```

Meanwhile, even in Python documentation `rglob` comes with the `sorted()` function, see [this example](https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob).

<img width="1178" alt="image" src="https://user-images.githubusercontent.com/10733514/164737108-f45a239a-d6f9-495d-8aad-52e3c5129b16.png">

**Suggested solution**

![image](https://user-images.githubusercontent.com/10733514/164734424-4ed5b6ce-6fff-4dda-8897-30dc02b25bc1.png)

I can create a pull request for that, if you want.

**Comment by makseq** [comment]
@mikbuch Could you please create a PR with your solution?

**Comment by dvwright** [comment]
Hi @mikbuch thank you for your submission! 

> IDs of the tasks have nothing to do with either filename of the original file, not the time when this file was created in the file system.

Would it make sense to sort by file creation time? Why are you choosing to sort by filename? 

Do you have an example scenario demonstrating the benefit of this behavior?

Thank you

**Comment by dvwright** [comment]
Confirmed, having this sorted is an improvement over the current behavior. I think by filename is fine. FWIW, to sort by created at timestamp would be ```key=os.path.getctime``` another possible option

example via the UI

before:
<img width="955" alt="unsorted" src="https://user-images.githubusercontent.com/191410/167939300-cfb9a80c-98d0-4247-b7de-916ac5cb389f.png">

after:
![sorted](https://user-images.githubusercontent.com/191410/167939353-6f482f5f-df6a-4dab-9781-51b34b374f03.png)

**Comment by makseq** [comment]
Thank you for the creation and the review of this PR!

**Comment by mikbuch** [comment]
Hi @dvwright , we chose sorting by filename for several reasons. First, it is default behavior in most file managers, so we wanted to keep the user (also understood as a programmer) consistent with what he/she sees in the file manager browsing files in the filesystem and what is then presented in the label-studio. Second, in our case file name contains some date and time information in a standardized form (not going into details). So in our use case the filename itself contains some kind of "creation time" (I know that this is something different from the time when the file was created in particular filesystem, nevertheless, we discussed this issue and we finally chose sorting by filename).

@makseq , I'm glad that I could help. You are doing an amazing job guys with label-studio, good luck!
